staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -141,12 +140,6 @@ func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 	serverConfig.OpenAPIConfig.Info.Title = "Wardle"
 	serverConfig.OpenAPIConfig.Info.Version = "0.1"
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.OpenAPIV3) {
```

Comment:
Why remove the below logic?

OpenAPIV3 is always enabled here.

Reply:
corrected.